### PR TITLE
Ytyp fixes

### LINKED ISSUE #202: YTYP exteriorVisibilityDepth property typo
the `exteriorVisibilityDepth` property intentionally has a typo in it in Codewalker, and is supposed to be `exteriorVisibiltyDepth`. This prevents the `exteriorVisibiltyDepth` from being properly imported/exported when using CodeWalker.

For Reference: [dexyfex/CodeWalker/issues/139](https://github.com/dexyfex/CodeWalker/issues/139)

### PR DESCRIPTION
**Pull request by CFox04**
resolves #202 and #201

### CODE REVIEW

File: ytyp/properties.py [hunk 1]
```diff
@@ -123,6 +123,10 @@ def get_room_to_name(self):
     def get_name(self):
         return f"{self.room_from_name} to {self.room_to_name}"
 
+    def update_audio_occlusion(self, context):
+        if self.audio_occlusion < 0:
+            self.audio_occlusion = ((2**31) - 1)
```
Comment (MoosheTV) [inline ytyp/properties.py:128]: is it intended for it to be the max value for a 32-bit *signed* integer? If not, you can do `2**32 - 1` for max value of an unsigned integer.

Comment (MoosheTV) [reply ytyp/properties.py:128]: if you also want some sort of rollover effect like overflowing/underflowing usually does, you could also do:

```py
if self.audio_occlusion < 0:
    self.audio_occlusion = self.audio_occlusion % 4294967296L # const value of 2 ** 32
```

Comment (Skylumz) [reply ytyp/properties.py:128]: @CFox04 so does this fix it or?

Comment (CFox04) [reply ytyp/properties.py:128]: > is it intended for it to be the max value for a 32-bit _signed_ integer? If not, you can do `2**32 - 1` for max value of an unsigned integer.

No that was not intended. I'll fix this.

Comment (CFox04) [reply ytyp/properties.py]: So the ``bpy.props.IntProperty`` does not seem to let me store a 32-bit unsigned integer even if I have the ``subtype="UNSIGNED"`` parameter. I just made it a string property and ensured it is the correct value in the update function.

**Pull request merged**